Fix russian locale.  Closes: #368655.

This aligns with the glibc definition in /usr/share/locale/locale.alias.
This only applies to Debian's glibc, see bug #62586 and glibc 2.2.5-4.

For the same glib reason, we cannot change ru and ru_RU away from ISO8859-5,
since that encoding is enshrined in /usr/share/i18n/locales/ru_RU.  Insert 
appropriate perjorative language here.

Suggestion: can the russian default be changed to UTF-8 instead of KOI8-R?  
This will make it easier for russians to fit in with the rest of the world.

Index: libx11/nls/locale.alias.pre
===================================================================
--- libx11.orig/nls/locale.alias.pre
+++ libx11/nls/locale.alias.pre
@@ -1265,7 +1265,7 @@
 portuguese.iso88591:				pt_PT.ISO8859-1
 romanian:					ro_RO.ISO8859-2
 rumanian:					ro_RO.ISO8859-2
-russian:					ru_RU.ISO8859-5
+russian:					ru_RU.KOI8-R
 serbocroatian:					sr_RS.UTF-8@latin
 sinhala:					si_LK.UTF-8
 slovak:						sk_SK.ISO8859-2
